payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -55,6 +53,61 @@ object PaymentsThemeConfig {
         val borderStrokeWidthSelected = 2.dp
     }
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    object Typography {
+        private val fontWeightBold: Int = FontWeight.Bold.weight
+        private val fontWeightMedium: Int = FontWeight.Medium.weight
+        private val fontWeightNormal: Int = FontWeight.Normal.weight
+        private val fontSizeMultiplier: Float = 1.0F
```

Comment:
What is the point of this?

Reply:
These are the public variables the eventual api will be able to change. For now they are just the defaults.